pkg/raft/log.go

```diff
@@ -42,10 +42,16 @@ type LogSnapshot struct {
 	storage LogStorage
 	// unstable contains the unstable log entries.
 	unstable LogSlice
+	// termCache contains a compressed entryID suffix of raftLog.
+	termCache *TermCache
 	// logger gives access to logging errors.
 	logger raftlogger.Logger
 }
 
+const (
+	termCacheSize = 10
+)
```

Comment:
We should probably start with 2 or 3.

```suggestion
const termCacheSize = 3
```

Would be good to have a small comment explaining what this is and why the constant is low.